Bizyhood/Core.php

```diff
@@ -1945,6 +1959,73 @@ function bizyhood_plugin_action_links( $links ) {
        return $links;
     }
     
+    
+    function load_rss() {
```

Comment:
Hey Eugene. By position I mean where it is placed in the URL, eg:

/business-events/rssfeed/
/business-events/vinnys-auto-repair-marlboro-nj-07746/rssfeed/

This will produce different query vars.